app/code/Magento/Catalog/Test/Mftf/Test/StorefrontAddProductWithBackordersAllowedOnProductLevelToCartTest.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontAddProductWithBackordersAllowedOnProductLevelToCartTest">
+        <annotations>
+            <features value="Catalog"/>
+            <stories value="Manage inventory, backorders"/>
+            <title value="Add Product to Cart, Backorders Allowed On Product Level"/>
+            <severity value="MAJOR"/>
+            <description value="Customer should be able to add products to Cart if product qty less or equal 0 and Backorders are allowed on Product level"/>
+            <group value="catalog"/>
+        </annotations>
+
+        <before>
+            <createData entity="SimpleProductInStockQuantityZero" stepKey="createProduct"/>
+            <actionGroup ref="AdminLoginActionGroup" stepKey="loginAsAdmin"/>
+        </before>
+
+        <after>
+            <actionGroup ref="AdminLogoutActionGroup" stepKey="logout"/>
+            <deleteData createDataKey="createProduct" stepKey="deleteProduct"/>
+        </after>
+
+        <actionGroup ref="NavigateToCreatedProductEditPageActionGroup" stepKey="openEditBundleProduct">
```

Comment:
@AnnaAPak Thank you for the pull request.
It looks good. 

I would like you to change the `stepKey` to more relevant. You can make it `openEditCreatedProduct` or `openEditSimpleProduct`